mhubert-pmn/Front-Url-Shortener
# [Feature] Create shorten url page

## Task
Create Shorten URL page

## Type of change

## Description
- Update content
- Link back to front

## Testing scenario
**/!\ The back must be running or you won't be able to test to sign in**
**/!\ At least one account must be created**"

In the terminal:
- Launch and build: `docker-compose up --build
- Go to `http://localhost:3000/`
- Enter the pseudo and password and click on the "Sign In" button: you should be logged in (check the value of the localStorage in your console)
- Go to `http://localhost:3000/raccourcir-une-url`
- Enter a link and click on the "Raccourcir" button
- A block should appear with the original link and the shorten link

## Trello card
[https://trello.com/c/xZl4pdDh](https://trello.com/c/xZl4pdDh)
